### ISSUE TITLE
Registrering mouse input

### ISSUE THREAD

**Issue by J0Beck** [robotnik-dev/character_camera_3d#3]
Hi,

It seems that the inputs for "Look_up" and the other "Look_[direction]" arent mapped to anything, but simple asks for signal strength.
Is this because the asset is primarily aimed toward use with controllers?

(Im new to godot, so sry if my question is noobish)

**Comment by robotnik-dev** [comment]
You are right they arent mapped to anything. You have to set it up in your project settings (Project Settings -> Input Map -> add new action for all 4 look_directions).
I tested only with arrow keys and controller input so yes, there is no mouse support but eventually i will add it.
